documentation/developer_setup.md

```diff
@@ -1,88 +1,74 @@
 # Developer Set Up
 
 * [Installing Go](#installing-go)
-* [Create a Client](#create-a-client)
-	* [Documenting Code](#documenting-code)
-	* [Constructors](#constructors)
-	* [Defining Methods](#defining-methods)
+* [Directory Structure](#directory-structure)
+* [Module Skeletop](#create-module-skeleton)
+* [Create SDK](#create-your-sdk)
 * [Write Tests](#write-tests)
 
 ## Installing Go
 
-The Azure-sdk-for-go team supports Go versions latest and latest-1, to see the exact versions we support you can check the pipeline defintions [here][pipeline_definitions]. The CI pipelines test the latest and latest-1 versions on both Windows and Linux virtual machines. If you do not already have Go installed, refer to this [workspace setup][workspace_setup] article for a more in depth tutorial on setting up your Go environment (there is also an MSI if you are developing on Windows at the [go download page][go_download]). After installing Go and configuring your workspace, fork the `azure-sdk-for-go` repository and clone it to a directory that looks like: `<GO HOME>/src/github.com/Azure/azure-sdk-for-go`.
+The Azure SDK for Go supports the latest two versions of Go. When setting up a new development environment, we recommend installing the latest version of Go per the [Go download page][go_download].
 
+### Configuring VSCode
 
-## Create a Client
+If you're using VSCode, install the Go extension for VSCode. This usually happens automatically when opening a .go file for the first time.
+See the [docs][vscode_go] for more information on using and configuring the extension.
 
-After you have the generated code from Autorest, the next step is to wrap this generated code in a "convenience layer" that the customers will use directly to interact with the service. Go is not an object-oriented language like C#, Java, or Python. There is no type hierarchy in Go. Clients and models will be defined as `struct`s and methods will be defined on these structs to interact with the service.
+## Directory Structure
 
-In other languages, types can be specifically marked "public" or "private", in Go exported types and methods are defined by starting with a capital letter. The methods on structs also follow this rule, if it is for use outside of the model it must start with a capital letter.
+Fork the `azure-sdk-for-go` repository and clone it to a directory that looks like: `<prefix-path>/Azure/azure-sdk-for-go`.
```

Comment:
Good link to include. How to fork/repo branching: https://github.com/Azure/azure-sdk/blob/main/docs/policies/repobranching.md

Reply:
Link added.